packages/interactivity/src/proxies/context.ts

```diff
@@ -0,0 +1,69 @@
+// TODO: Use the proxy registry to avoid multiple proxies on the same object.
+
+// Store the context proxy and fallback for each object in the context.
+const contextObjectToProxy = new WeakMap();
+const contextProxyToObject = new WeakMap();
+const contextObjectToFallback = new WeakMap();
+
+const descriptor = Reflect.getOwnPropertyDescriptor;
+
+const contextHandlers: ProxyHandler< object > = {
+	get: ( target: object, k: string ) => {
```

Comment:
It might be better to use `key` instead of `k` here to follow the same nomenclature as in the rest of the handlers.

Reply:
Right, I'll do that.